desk/lib/dm.hoon

```diff
@@ -1,5 +1,5 @@
-/-  c=chat, d=channels
-/+  mp=mop-extensions
+/-  c=chat, d=channels, meta
+/+  mp=mop-extensions, cu=channel-utils
```

Comment:
Surprising that this file needs to be changed even though chat continues using the old types. Not sure I understand why.

And as far as I can tell, you don't actually use the `meta` import, do you?